src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -9089,6 +9089,45 @@ public class C2
                 Diagnostic(ErrorCode.ERR_BadAttributeArgument, "C.M(null)").WithLocation(20, 6));
         }
 
+        [Fact]
+        [WorkItem(44049, "https://github.com/dotnet/roslyn/issues/44049")]
+        public void MemberNotNullAnnotationCrashes()
```

Comment:
>MemberNotNullAnnotationCrashes [](start = 20, length = 30)

Consider moving to NullableReferenceTypesTests.cs since most (all?) of the `[MemberNotNull]` tests are there.

Also, consider renaming since the test since the test should not crash. Perhaps something like `MemberNotNull_InstanceMemberOnStaticMethod`.